### DISCUSSION TITLE
what does "Video camera P2P Stream Start P2P Stream" do in Camera integration do

### DISCUSSION THREAD

**Discussion by rpeng6** [XiaoMi/ha_xiaomi_home#350] [Q&A]
Just installed the Xiaomi Home Integration really love it and everything working good but i am not sure what this "Video camera P2P Stream Start P2P Stream" button do in the camera integration do ? does it provide a local stream so it could be viewed in Home assist if so could more instruction be provided.
below is the camera info 
chuangmi.camera.ip029a
![image](https://github.com/user-attachments/assets/851e6945-dd63-4031-b6db-82d6414ef4a0)

**Comment by JimmyKmi** [comment]
This is an automation that sends the screen to Mi Home Screen and Xiao Ai with a display.

It does not support sending the screen to HomeAssistant.

https://github.com/XiaoMi/ha_xiaomi_home/issues/40
https://github.com/XiaoMi/ha_xiaomi_home/issues/62
https://github.com/XiaoMi/ha_xiaomi_home/discussions/128
https://github.com/XiaoMi/ha_xiaomi_home/issues/160
https://github.com/XiaoMi/ha_xiaomi_home/discussions/292
https://github.com/XiaoMi/ha_xiaomi_home/discussions/232
https://github.com/XiaoMi/ha_xiaomi_home/issues/309

**Reply by corrado-c** [reply]
but is it possible for google home and alexa, can't it be used for homeassistant?

**Comment by corrado-c** [comment]
I'm interested in the topic, I was looking at the way to see Xiaomi cam images with HomeAssistant, will it be available in the future?
